### PR TITLE
Content migration batch 2

### PR DESCRIPTION
**Pull request by mattletford**
Migrating content

From: https://developers.google.com/web/updates/2015/11/presentation-api
To: https://developers.google.com/blog/presentation-api/

From: https://developers.google.com/web/updates/2017/03/chrome-58-deprecations
To: https://developers.google.com/blog/chrome-58-deprecations/

From: https://developers.google.com/web/updates/2017/04/chrome-59-deprecations
To: https://developers.google.com/blog/chrome-59-deprecations/

From: https://developers.google.com/web/updates/2017/06/chrome-60-deprecations
To: https://developers.google.com/blog/chrome-60-deprecations/

From: https://developers.google.com/web/updates/2017/08/chrome-61-deprecations
To: https://developers.google.com/blog/chrome-61-deprecations/

@rachelandrew

### CODE REVIEW

File: site/_data/authorsData.json [hunk 1]
```diff
@@ -398,5 +398,10 @@
     "github": "victortan",
     "twitter": "victortan",
     "image": "image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/j7BoDYMJWpcjTNyhbnYu.jpeg"
+  },
+  "josephmedley": {
```
Comment (rachelandrew) [inline site/_data/authorsData.json:402]:
```suggestion
  "joemedley": {
```

File: site/_data/i18n/authors.yaml [hunk 2]
```diff
@@ -359,4 +359,8 @@ victortan:
     en: 'Victor Tan'
   description:
     en: 'Software Engineer, Google Chrome'
-    
+josephmedley:
```
Comment (rachelandrew) [inline site/_data/i18n/authors.yaml:362]:
```suggestion
joemedley:
```

File: site/_data/i18n/authors.yaml [hunk 3]
```diff
@@ -359,4 +359,8 @@ victortan:
...
+  title:
+    en: 'Joseph Medley'
```
Comment (rachelandrew) [inline site/_data/i18n/authors.yaml:364]:
```suggestion
    en: 'Joe Medley'
```

File: site/en/blog/chrome-58-deprecations/index.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+[implemented in Chrome 45](https://www.chromestatus.com/feature/6190642178818048).
+The names of these properties were changed in the spec in mid 2016. Chrome
+implemented the
+[new names in Chrome 55 and Chrome 56](https://www.chromestatus.com/feature/6390764217040896).
+Console deprecation warnings were also implemented.
+
+In Chrome 58, the old property names are being removed. The affected properties
+and their new names are shown below.
+
+| Removed Property | Current Name    |
```
Comment (rachelandrew) [inline site/en/blog/chrome-58-deprecations/index.md:88]: This needs to be an HTML table as in the handbook.

File: site/en/blog/chrome-59-deprecations/index.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Deprecations and Removals in Chrome 59
+description: >
+  A round up of the deprecations and removals in Chrome 59 to help you plan.
+authors:
+  - josephmedley
```
Comment (rachelandrew) [inline site/en/blog/chrome-59-deprecations/index.md:7]:
```suggestion
  - joemedley
```

File: site/en/blog/chrome-60-deprecations/index.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Deprecations and Removals in Chrome 60
+description: >
+  A round up of the deprecations and removals in Chrome 60 to help you plan. In this version, security improvements, further webkit deprecations, and more.
+authors:
+  - josephmedley
```
Comment (rachelandrew) [inline site/en/blog/chrome-60-deprecations/index.md:7]:
```suggestion
  - joemedley
```

File: site/en/blog/chrome-61-deprecations/index.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Deprecations and Removals in Chrome 61
+description: >
+  A round up of the deprecations and removals in Chrome 61 to help you plan. In this version, security improvements, further webkit deprecations, and more.
+authors:
+  - josephmedley
```
Comment (rachelandrew) [inline site/en/blog/chrome-61-deprecations/index.md:7]:
```suggestion
  - joemedley
```

File: site/en/blog/presentation-api/index.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+
+The `currentMedia` variable here is a `chrome.cast.media.Media` object, which can be used for controlling playback:
+
+```js
+function playMedia() {
+  currentMedia.play(null, success, error);
+}
+```
+
+    // ...
```
Comment (rachelandrew) [inline site/en/blog/presentation-api/index.md:132]: this looks odd

File: site/en/blog/presentation-api/index.md [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+```js
+function sessionListener(e) {
+  session = e;
+  if (session.media.length !== 0) {
+    onMediaDiscovered('onRequestSessionSuccess', session.media[0]);
+  }
+}
+```
+
+<a id="disable-remote-playback">
```
Comment (rachelandrew) [inline site/en/blog/presentation-api/index.md:153]: I'm not sure what this is

**Review by rachelandrew — CHANGES_REQUESTED** [review]

File: site/_data/authorsData.json (hunk 1 above)
Comment (mattletford) [reply site/_data/authorsData.json]: @rachelandrew removed duplicate

File: site/_data/i18n/authors.yaml (hunk 2 above)
Comment (mattletford) [reply site/_data/i18n/authors.yaml]: @rachelandrew removed duplicate

File: site/_data/i18n/authors.yaml (hunk 3 above)
Comment (mattletford) [reply site/_data/i18n/authors.yaml]: @rachelandrew removed duplicate

File: site/en/blog/chrome-58-deprecations/index.md (hunk 4 above)
Comment (mattletford) [reply site/en/blog/chrome-58-deprecations/index.md]: @rachelandrew changed to HTML table

File: site/en/blog/chrome-59-deprecations/index.md (hunk 5 above)
Comment (mattletford) [reply site/en/blog/chrome-59-deprecations/index.md]: @rachelandrew removed duplicate

File: site/en/blog/chrome-60-deprecations/index.md (hunk 6 above)
Comment (mattletford) [reply site/en/blog/chrome-60-deprecations/index.md]: @rachelandrew removed duplicate

File: site/en/blog/chrome-61-deprecations/index.md (hunk 7 above)
Comment (mattletford) [reply site/en/blog/chrome-61-deprecations/index.md]: @rachelandrew removed duplicate

File: site/en/blog/presentation-api/index.md (hunk 8 above)
Comment (mattletford) [reply site/en/blog/presentation-api/index.md]: @rachelandrew removed

File: site/en/blog/presentation-api/index.md (hunk 9 above)
Comment (mattletford) [reply site/en/blog/presentation-api/index.md]: @rachelandrew removed

**Review by rachelandrew — APPROVED** [review]

**Review by rachelandrew — CHANGES_REQUESTED** [review]
this is conflicting again, possibly due to another merge.

**Comment by mattletford** [comment]
@rachelandrew resolved conflicts

**Pull request closed without merging**